src/res/failcode.en.json

```diff
@@ -31,6 +31,7 @@
     "bdBranchExBP.bdblock": "Lines are branched out of the point.",
     "bdCountLt3BP.bdblock": "A point has two or less lines.",
     "bdCross": "There is a crossing border line.",
+    "bdCross.squarejam": "Some region borders form a four-way intersection.",
```

Comment:
This seems like a philosophy question of fewest possible error codes vs. more error codes that more closely match the puzzle descriptions. I was leaning towards the latter, since it seems like error codes are cheap in the grand scheme of things